shared/src/main/scala/squants/market/Money.scala

```diff
@@ -377,6 +384,9 @@ final class Money private (val amount: BigDecimal)(val currency: Currency)
     * @return Money
     */
   def mapAmount(f: BigDecimal => BigDecimal) = currency(f(amount))
+
+
```

Comment:
Could you remove the extra lines?

Reply:
I fixed the formatting back to the original(style).